src/components/details/styles.jsx

```diff
@@ -37,7 +33,7 @@ const CarDetailsStyles = styled.div`
       }
     }
 
-    .car-text {
+    .carText {
```

Comment:
You changed these names here, but the class names on the components need to be changed.